WordPress/Classes/Stores/ActivityStore.swift

```diff
@@ -301,4 +368,29 @@ private extension ActivityStore {
 
         return ActivityServiceRemote(wordPressComRestApi: api)
     }
+
+    private func mediumString(from date: Date, adjustingTimezoneTo site: JetpackSiteRef) -> String {
```

Comment:
I'm not tooooo happy about this living here, especially given that it's pretty much replicated in the `ActivityListViewModel`. I thought letting the `Store` return this, so the `ViewModel` could just consume it, but... having something called `Store` return a `DateFormatter` feels real wrong to me.

I really want to keep posting notices as a responsibility of this class, so I'm sorta out of ideas how to make it nicer — suggestions definitely very welcome!